db/seeds.rb

```diff
@@ -33,9 +33,9 @@
 		phone: "206-723-1076",
 		website: "www.ugm.org"
 	})
-SeattleUnionGospelMission.institution_detail = InstitutionDetail.new({
-		hours: "No Hours Set"
-	})
+# SeattleUnionGospelMission.institution_detail = InstitutionDetail.new({
+# 		hours: "No Hours Set"
+# 	})
```

Comment:
Yeah I suppose its becoming outdated. I think we should still keep it in though. It gives a good basis for basic functionality. If people are using their own db, they don't have to use the seeded stuff